### ISSUE TITLE
Fix some issues in the main file

### ISSUE THREAD

**Issue by blessingblockchain** [Rakesh9100/ML-Project-Drug-Review-Dataset#98]
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

Importing from bs4 import BeautifulSoup, MarkupResemblesLocatorWarning raises a ImportError as MarkupResemblesLocatorWarning is not part of the bs4 library. You can remove MarkupResemblesLocatorWarning from the import statement.

The line train_imp[["rating"]] = train_imp[["rating"]].astype("float16") and test_imp[["rating"]] = test_imp[["rating"]].astype("float16") should be modified to train_imp["rating"] = train_imp["rating"].astype("float16") and test_imp["rating"] = test_imp["rating"].astype("float16"), respectively. Since you are selecting a single column, the double brackets are not necessary.

The code snippet does not show the import statements for the libraries used, such as seaborn, matplotlib.pyplot, imblearn, keras, and tensorflow. Make sure to include the necessary import statements before using these libraries.

There is a missing closing parenthesis in the line cm = confusion_matrix(Y_test, logi_test, labels=logi.classes_) when plotting the confusion matrix. Add a closing parenthesis at the end of the line.   assign me this issue to fix under GSSoC @r

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

**Comment by Rakesh9100** [comment]
MarkupResemblesLocatorWarning is the library to just handle and minimise the warning so it can't raise any import error.
You can install every libraries mentioned in `requirements.txt` to avoid any library error in the code.
So no issues as for now and hence closing the issue. Pls work on other good issues.
